packages/block-library/src/button/edit.native.js

```diff
@@ -185,18 +188,19 @@ class ButtonEdit extends Component {
 			);
 		}
 		return (
-			wrapperProps?.style?.backgroundColor ||
+			wrapperProps?.style?.backgroundColor || colorProps.style?.backgroundColor ||
```

Comment:
Hmmm, I think we could remove using of `wrapperProps` here since we use `colorProps` instead

Reply:
Will remove then